resources/random.go

```diff
@@ -0,0 +1,13 @@
+package rps
```

Comment:
Should be called resources

```suggestion
package resources
```